### ISSUE TITLE
Design & visual styling review of FEC Records 

### ISSUE THREAD

**Issue by jenniferthibault** [18F/fec-style#614]
Review:
- FEC Record article page
- FEC Record item in Latest updates feed 

Access pages in production while signed into the CMS.

Review & implemented improvements should happen before release

**Comment by AmyKort** [comment]
Comments from reviewers:

1) I’m not crazy about how the first paragraph is bigger font than the rest for each article.

2) We’re kind of losing the “sections” – unless that is what will be in the “subjects” drop down menu. Or will the subjects be what is in the current Record archive page at http://www.fec.gov/pages/fecrecord_redesign/fecrecord.shtml? I am slowly populating that page but as you see I am only back to 2004. 

3) We will need to think about the graphics we use in some articles (mainly outreach and statistics). What works for current fec.gov might not work in the beta site. For example, that corporate webinar graphic looks a little strange and big to me as placed. They used our placement for the year-end webinar and the new Chairman articles, but again the graphics look too big on the beta site. It might be helpful for 18F to give us size preferences or recommendations for graphics.

4) This is more Record decision-making but going forth I suggest that we ditch footnotes in articles. 

5) Will we be retaining the annual subject indexes at http://www.fec.gov/pages/record.shtml? 

6) For the articles that are 2011 and earlier – Those are coming from PDF articles so I suspect they are drawing that first sentence from the archive page listing. It’s just repeating the title twice and it kind of looks strange to do that.

7) Within article, use thumbnails linked to full size graphics (where appropriate), but do not include graphics in the “Latest Updates” list.

8) Add “[Read more]” link to make it clear how to access the full article.

9) Use initial cap’s for categories (e.g., Outreach, Advisory Opinions, etc.)

10) Will we manually enter the “Related” content at the bottom of each article or will it be generated based on tags . . .?

11) Instead of marking articles From: Information Division, can we say FEC Record ?

12) The email option next to the name of the author gives the appearance of suggesting readers contact that particular individual. Can we make it clear instead that there is a general email address for questions about campaign finance law and reporting requirements/

![latest updates](https://cloud.githubusercontent.com/assets/13237823/22041083/6c4f1190-dcd3-11e6-9ca1-421362496a4d.jpg)

**Comment by jenniferthibault** [comment]
I'm adding a few thoughts from my own review so that I won't lose track of them, but will come back to sort and respond to the questions and points above. Thanks so much for collecting these, Amy!

- Missing "Subject" categories in filter
  - ![screen shot 2017-01-17 at 9 18 12 pm](https://cloud.githubusercontent.com/assets/11636908/22048372/82c4e0ea-dcfa-11e6-8df2-ae1d019ea3cd.png)

- Implement cropped preview images in feed instead of full-size images. 
_Previously filed and documented in issue: https://github.com/18F/fec-cms/issues/549_
_Currently, images appear at whatever their original size is:_
  - ![screen shot 2017-01-17 at 9 08 10 pm](https://cloud.githubusercontent.com/assets/11636908/22048389/ad049166-dcfa-11e6-850c-6fb1ba9a57fd.png)
  - ![screen shot 2017-01-17 at 9 08 17 pm](https://cloud.githubusercontent.com/assets/11636908/22048390/ad067bf2-dcfa-11e6-9fab-5295d160a52f.png)

- If Author doesn't have position title listed, display nothing instead of `N/A`
  - <img width="237" alt="screen shot 2017-01-17 at 6 21 30 pm" src="https://cloud.githubusercontent.com/assets/11636908/22048438/08e81f98-dcfb-11e6-8236-ee6105846639.png">


- Consider replacing old speaker icon with new icon style
  - <img width="385" alt="screen shot 2017-01-17 at 6 21 58 pm" src="https://cloud.githubusercontent.com/assets/11636908/22048513/8b0f6b70-dcfb-11e6-9a65-353ec5972b4e.png">

**Comment by noahmanger** [comment]
Just made several of these changes in https://github.com/18F/fec-cms/issues/752 . I have separate thoughts on the image issue which I'll post there.

**Comment by jenniferthibault** [comment]
Going to close this since the review is complete, but next steps are to file issues for the outstanding feedback we should address before release.

**Comment by jenniferthibault** [comment]
Oops, that was a little early. I should have probably left the issue open so we don't lose it, but move it to "Done"

**Comment by jenniferthibault** [comment]
> 1. I’m not crazy about how the first paragraph is bigger font than the rest for each article.

We've heard this feedback on other items in the Latest Updates as well. That style has been removed from the Record templates. It will be removed from other templates by https://github.com/18F/fec-cms/issues/788
____
> 2. We’re kind of losing the “sections” – unless that is what will be in the “subjects” drop down menu... 

These sections are the categories users can select from the drop-down, which allows you to filter down to viewing all reports for that category. This was implemented in the last release.
____
> 3. We will need to think about the graphics we use in some articles (mainly outreach and statistics). What works for current fec.gov might not work in the beta site. 

Definitely agreed! Issue filed for this: https://github.com/18F/fec-style/issues/622
____
> 5. Will we be retaining the annual subject indexes at http://www.fec.gov/pages/record.shtml?

The Latest updates page will allow users to filter by both date and category, so I believe the same functionality should exist, even though it will look a little different.
____
> 6. For the articles that are 2011 and earlier – Those are coming from PDF articles [...] It’s just repeating the title twice and it kind of looks strange to do that.

Issue filed for this: https://github.com/18F/fec-cms/issues/790
____
> 7. Within article, use thumbnails linked to full size graphics (where appropriate), but do not include graphics in the “Latest Updates” list.

Issue discussing image treatment and approach: https://github.com/18F/fec-cms/issues/549
____
> 8. Add “[Read more]” link to make it clear how to access the full article.

Done! This was a planned feature and I'm happy to report that will be implemented in this week's release: https://github.com/18F/fec-cms/pull/752
____
> 9. Use initial cap’s for categories (e.g., Outreach, Advisory Opinions, etc.)

Updated to sentence case for consistency and will be part of this week's release: https://github.com/18F/fec-cms/pull/752
____
> 10. Will we manually enter the “Related” content at the bottom of each article or will it be generated based on tags . . .?

Related posts are generated programmatically by date, but can be overwritten manually. Related section appears to have been set to "data" by default, but should probably be empty
____
> 11. Instead of marking articles From: Information Division, can we say FEC Record ?

The biggest concern is that this change would be duplicative, and the feed item would say "FEC Record" in two places very close to one another. The secondary concern is that this is the only place that we say who publishes this content. Otherwise the Info Div piece is totally lost for readers. 
____
> 12. The email option next to the name of the author gives the appearance of suggesting readers contact that particular individual. Can we make it clear instead that there is a general email address for questions about campaign finance law and reporting requirements.

We can easily implement this to display any email address and contact info, but we aren't sure what it should say. New issue to track this: https://github.com/18F/fec-cms/issues/791

____
> "At first I thought I'd be able to click on [ subject tag] to go to the Record newsletter page or the Outreach section of the Record, but it seems to just take me back to the latest updates list"

We've changed the interaction so that when you select a tag, instead the page skips the Title content and anchors the dropdowns at the top of the screen. That way, we hypothesize that users can tell the content changed more easily.
____
Other issues filed: 
- [To avoid confusing previews in the Latest Updates feed, cut off FEC Record text preview text after first sentence #789](https://github.com/18F/fec-cms/issues/789)


____ 
Closing this issue since each item can now be tracked independently

**Comment by AmyKort** [comment]
Thank you @jenniferthibault !

**Comment by dorothyyeager** [comment]
Thank you @jenniferthibault - going through it all now!
